eth/gasprice/feehistory_test.go

```diff
@@ -62,7 +62,7 @@ func TestFeeHistory(t *testing.T) {
 		oracle := NewOracle(backend, config)
 
 		first, reward, baseFee, ratio, err := oracle.FeeHistory(context.Background(), c.count, c.last, c.percent)
-
+		backend.chain.Stop()
```

Comment:
It feels weird to call chain stop here. Maybe we should define a teardown function for the backend and call `defer backend.teardown` after initializing backend?

Reply:
I agree about calling `backend.teardown()` instead of reaching in to the `chain`, but I'd rather call it here directly rather than `defer` it.